compiler/src/iree/compiler/API/python/CMakeLists.txt

```diff
@@ -169,6 +162,43 @@ if(TARGET lld)
   )
 endif()
 
+# Install shared libraries that the extension depends on. This uses
+# CMake's defer feature to evaluate the install directive once everything
+# has been evaluated (because there is no guarantee that this file evaluates
+# before the API libraries are defined). While deferred calls are generally
+# fragile, this install is purely a leaf feature (with no other calls
+# depending on its sequencing), so this use is okay.
+# We defer it to the compiler/ directory so that cmake_install.cmake targets
+# are all self contained to the compiler, which is the most common spanning
+# parent.
+cmake_language(EVAL CODE "
```

Comment:
:scream:

Reply:
Yeah. Right reaction. But I copied this from some code that I think Scott wrote, so I can't claim responsibility for this hack